game_start/game.js

```diff
@@ -1,26 +1,26 @@
 const btnPlay = document.querySelector('.startGame');
 const addCards = document.querySelector('.cards');
-let cards = [...document.querySelectorAll('.card')];
+let cards = document.querySelectorAll('.card');
 const moves = document.querySelector('.moves');
 const cardsArray = [
-	'pumpkin',
```

Comment:
Te wszystkie poprawki formatowania są z automatu robione przez mojego VS Code. Jeżeli jeszcze nie korzystasz to bardzo polecam (https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode). Do tego możesz w ustawieniach zaznaczyć format on save i za każdym savem będzie poprawiało kod, żeby był dobrze sformatowany.